Source/AlchemyLanguageV1/AlchemyLanguageV1.swift

```diff
@@ -0,0 +1,1956 @@
+    /**
+     Calculates the Taxonomy of given content.
+     
+     - parameter text:    a Text document
+     - parameter failure: a function executed if the call fails
+     - parameter success: a function executed with Taxonomy information
+     */
+    public func getRankedTaxonomyText(
+        text: NSURL,
+        failure: (NSError -> Void)? = nil,
+        success: Taxonomies -> Void)
+    {
+        // construct body
+        let body = try? buildBody(text, html: false)
+        
+        // construct query paramerters
+        var queryParams = [NSURLQueryItem]()
+        
+        queryParams.append(NSURLQueryItem(name: "apikey", value: apiKey))
+        queryParams.append(NSURLQueryItem(name: "outputMode", value: "json"))
+        
+        // construct request
+        let request = RestRequest(
+            method: .POST,
+            url: serviceUrl + "/text/TextGetRankedTaxonomy",
+            contentType: "application/x-www-form-urlencoded",
+            messageBody: body,
+            queryParameters: queryParams
+        )
+        
+        // execute request
+        Alamofire.request(request)
+            .responseObject(dataToError: dataToError) {
+                (response: Response<Taxonomies, NSError>) in
+                switch response.result {
+                case .Success(let taxonomies): success(taxonomies)
+                case .Failure(let error): failure?(error)
+                }
+        }
+    }
+    
+    /**
+     Extracts the Raw Text of given content.
+     
+     - parameter url:     the URL of the content
+     - parameter failure: a function executed if the call fails
+     - parameter success: a function executed with Raw Text information
+     */
+    public func getRawTextURL(
+        url: String,
+        failure: (NSError -> Void)? = nil,
+        success: DocumentText -> Void)
+    {
+        // construct request
+        let request = RestRequest(
+            method: .POST,
+            url: serviceUrl + "/url/URLGetRawText",
+            contentType: "application/x-www-form-urlencoded",
+            queryParameters: [
+                NSURLQueryItem(name: "url", value: url),
+                NSURLQueryItem(name: "apikey", value: apiKey),
+                NSURLQueryItem(name: "outputMode", value: "json")
+            ]
+        )
+        
+        // execute request
+        Alamofire.request(request)
+            .responseObject(dataToError: dataToError) {
+                (response: Response<DocumentText, NSError>) in
+                switch response.result {
+                case .Success(let docText): success(docText)
+                case .Failure(let error): failure?(error)
+                }
+        }
+    }
+    
+    /**
+     Extracts the Raw Text of given content.
+     
+     - parameter html:    a HTML document
+     - parameter url:     a reference to where the HTML is located
+     - parameter failure: a function executed if the call fails
+     - parameter success: a fu
```

Comment:
Hehe, both. The Swagger docs often contain incomplete information. For example several calls return a `totalTransactions` field that isn't mentioned once. I've omitted `status` on most of the objects, as it seems to me to only be useful to check for errors and not really useful to hold on to,